crates/ruff_linter/src/rules/flake8_bugbear/rules/useless_comparison.rs

```diff
@@ -32,7 +32,7 @@ impl Violation for UselessComparison {
     #[derive_message_formats]
     fn message(&self) -> String {
         format!(
-            "Pointless comparison. This comparison does nothing but waste CPU instructions. \
+            "Pointless comparison. This comparison does nothing. Did you mean to assign a value? \
```

Comment:
I think I'd suggest removing "This comparison does nothing.", it feels redundant.